app/jobs/operation_job.rb

```diff
@@ -3,53 +3,58 @@ class OperationJob < ApplicationJob
 
```

Comment:
We need a class comment on this one too please. Explain the difference between Operation and OperationJob while it's still fresh in our heads :D